src/components/Navigation/components/Item/Item.tsx

```diff
@@ -88,20 +99,34 @@ export default class Item extends React.Component<Props, State> {
       onClick,
       accessibilityLabel,
       iconBody,
+      new: isNew,
+      polaris: {intl},
       selected: selectedOverride,
     } = this.props;
 
     const {location, onNavigationDismiss} = this.context;
 
     const tabIndex = disabled ? -1 : 0;
 
-    const badgeMarkup = badge && (
-      <div className={styles.Badge}>
-        <Badge status="new" size="small">
-          {badge}
-        </Badge>
-      </div>
-    );
+    const hasNewChild =
+      subNavigationItems.filter((subNavigationItem) => subNavigationItem.new)
+        .length > 0;
+
+    const indicatorMarkup = hasNewChild ? (
+      <span className={styles.Indicator}>
+        <Indicator pulse />
+      </span>
+    ) : null;
+
+    const badgeMarkup =
+      badge ||
```

Comment:
Is this a regression? Looks like `badgeMarkup` now gets the bare value of `badge`. It should still get wrapped in markup.

Reply:
Oh this is just bad formatting, the bare badge value was just to use as a conditional for the rendering. I will change it to a ternary to match the other conditional markup.